BREAKING-CHANGES-EXAMPLES.md

```diff
@@ -82,11 +82,9 @@ These examples are automatically generated from unit tests.
 [adding a new required property under AllOf in response body is not breaking](checker/checker_breaking_property_test.go?plain=1#L432)  
 [adding a new required read-only property in request body is not breaking](checker/checker_breaking_property_test.go?plain=1#L486)  
 [adding a non-existent required property in request body is not breaking](checker/checker_breaking_property_test.go?plain=1#L294)  
-[adding a tag is not breaking with "api-tag-removed" check](checker/checker_not_breaking_test.go?plain=1#L276)  
```

Comment:
@reuvenharrison I was reserving the `CL`comments to checker_test files, and keeping the ones in checker_not_breaking as BC so that we keep this doc up to date. 

If you look at the entries below we are ending up duplicating what are changelog entries. LMK what you think and if I can revert this commit.